qutebrowser/config/configdata.py

```diff
@@ -301,6 +301,10 @@
     )),
 
     ('completion', sect.KeyValue(
```

Comment:
First I thought `completion` was a weird place to put this in... but I don't think there's anywhere where it'd fit better. :)